db/migrate/20230920153421_add_needs_apple_delivery_to_episode.rb

```diff
@@ -0,0 +1,29 @@
+class AddNeedsAppleDeliveryToEpisode < ActiveRecord::Migration[7.0]
+  def change
+    add_column :episodes, :needs_apple_delivery, :boolean, default: true
```

Comment:
Just curious - why track on the `Episode` vs the `Apple::Episode`?  Or I guess that's not a table ... so the `Apple::PodcastContainer`?  It seems like you've already got some logic in there to check if it needs re-delivery.  Do you need to set this flag when that container doesn't exist yet or something?

(I'm thinking that querying for `Episode.where(needs_apple_delivery: true)` wouldn't exactly be accurate, since that returns shows that don't do delegated delivery.  Where `Apple::PodcastContainer.where(needs_apple_delivery: true)` would.)

Reply:
Really good point, let's move this off the episode.